src/software/thunderscope/robot_diagnostics/chicker_widget.py

```diff
@@ -21,7 +21,7 @@ def __init__(self, proto_unix_io):
         to be sent to the robots.
 
         NOTE: The powerboards run in regulation mode, which means that they are
-        always charged and do not need to be explicitly charged. 
+        always charged and do not need to be explicitly charged.
```

Comment:
There might be a mismatch in versions of files, or this might also be a problem in the other PR as well, but I'm getting this error when I try to launch with diagnostics.
```
File "/home/nima/.cache/bazel/_bazel_nima/d4d8c6a2a61c53de6e279fde073469b4/execroot/__main__/bazel-out/k8-fastbuild/bin/software/thunderscope/thunderscope_main.runfiles/__main__/software/thunderscope/robot_diagnostics/chicker_widget.py", line 42, in __init__
    self.push_button_box, self.push_buttons = common_widgets.create_button(
AttributeError: module 'software.thunderscope.common.common_widgets' has no attribute 'create_button'
```

Reply:
That's weird, I noticed that on master the method is create_button, but create_buttons seems to be the correct one. I've commited that change so should be fixed now